contribution/vagrant/Vagrantfile

```diff
@@ -83,6 +83,9 @@ Vagrant.configure("2") do |config|
       # skip to set up the development env
     else
       # install base dependencies
+
```

Comment:
Small nit, let's not add unnecessary new lines. Rest LGTM. Thanks for the updates.

Reply:
Deleted Empty lines , Done.